README.md

```diff
@@ -1,26 +1,83 @@
 # OpenShift MySQL image
 
-This repository contains Dockerfiles for MySQL images for OpenShift. Users can choose between RHEL and CentOS based images.
-
+This repository contains Dockerfiles for MySQL images for OpenShift.
+Users can choose between RHEL and CentOS based images.
 
 # Installation and Usage
-
 Choose between CentOS7 or RHEL7 base image:
 
 *  **RHEL7 base image**
```

Comment:
maybe we should change "base" to something else? "RHEL7 image"? The "base" sounds like a sti-base